utils/ntfs-3g/Makefile

```diff
@@ -22,11 +21,8 @@ PKG_FIXUP:=autoreconf
 PKG_INSTALL:=1
 PKG_BUILD_PARALLEL:=1
 
-# release contains fuseext/int hint
-PKG_RELEASE:=$(PKG_RELEASE)$(if $(CONFIG_PACKAGE_NTFS-3G_USE_LIBFUSE),-fuseext,-fuseint)
-
 # define build dir, respect fuseext/int
-PKG_BUILD_DIR:= $(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-$(PKG_RELEASE)
+PKG_BUILD_DIR:= $(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
```

Comment:
that's default, no?

Reply:
Packages do not append RELEASE to build dir names. However, if there are variants, which in this package they are not, then some other naming scheme would be necessary.  There was never any intention to have multiple variants of this package building at the same time - so no variant naming is necessary.

Reply:
Sorry - answered wrong question. You are correct, this turns out to be the default if no variant, etc.

```
PKG_BUILD_DIR ?= $(BUILD_DIR)/$(if $(BUILD_VARIANT),$(PKG_NAME)-$(BUILD_VARIANT)/)$(PKG_NAME)$(if $(PKG_VERSION),-$(PKG_VERSION))
```